### DISCUSSION TITLE
ビルドに必要なメモリが増えている件 / About the increase in memory required for builds

### DISCUSSION THREAD

**Discussion by tamaina** [misskey-dev/misskey#13415] [General]
ビルドに必要なメモリが増えた件

### TODO？（誰かやってほしい）

- [ ] 現在のメモリ消費量の確認
- [ ] 解決策の議論
  * ビルドコマンドの見直し
  * ドキュメントの見直し (使用メモリ量についての言及を見直す、Dockerイメージ利用の推奨)

**Comment by syuilo** [comment]
ビルド不要にしたい

**Reply by tamaina** [reply]
バックエンドのts含めてかしら

（viteをビルド不要にするのは効果ありそうだし簡単そうだけど、バックエンドはswcかesbuildでtsに直してやった方がいいと思っている）

**Reply by syuilo** [reply]
ビルドはするけどユーザー側としては不要にしたい

**Reply by tamaina** [reply]
Dockerですでに達成されてる感じはある

**Reply by kakkokari-gtyih** [reply]
リリース時にビルドを回して、ビルドされた成果物をtarで配布できるようにすればいいのかも
